servicepulse/intro-archived-messages.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Archived Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows un-archiving archived failed messages
+tags:
+- ServicePulse
+reviewed:  
+---
+
+### Introduction
+
+In SerivcePulse 1.5.0 and higher screens to manage archived messages have been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal. 
```

Comment:
typo -  `SerivcePulse 1.5.0`
It would read better if it was, `In ServicePulse Versions 1.5.0 and above`